file_paths.md

```diff
@@ -204,6 +217,20 @@ user or group.
             run                               # :rundir                      same
                 agent.pid                     # :pidfile
 
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        etc *
+            pxp-agent.cfg                     # pxp-agent configuration file
+            modules.d *                       # directory where pxp-agent modules are stored
+                puppet.cfg *                  # ../../var/modules/puppet configuration
+        var *
+            log
+                pxp-agent.log                 # enabled by default
+            modules *
```

Comment:
I will leave `C:\ProgramData\PuppetLabs\pxp-agent\etc\modules` as the directory for module config files.

@kylog , the PR is currently specifying `C:\ProgramData\PuppetLabs\pxp-agent\var\modules` as the directory for module binaries on Windows. I will follow @Iristyle 's suggestion and put them under `\Program Files\'.

Do we agree on `C:\Program Files\Puppet Labs\Puppet\pxp-agent\modules`?